bundler.d/assets.rb

```diff
@@ -1,7 +1,10 @@
 gem 'jquery-rails', '~> 3.1'
 gem 'jquery-turbolinks', '~> 2.1'
 gem 'jquery-ui-rails', '< 5.0.0'
-gem 'bootstrap-sass', '3.0.3.0'
+gem 'autoprefixer-rails', '~> 5.2'
+gem 'sprockets', '~> 2.12'
```

Comment:
AFAIU  you need at least 2.8 to work with sass-rails https://rubygems.org/gems/sass-rails